bismuth.spec

```diff
@@ -29,19 +30,67 @@ them via keyboard, just like in classical tiling window managers
 
 %files
 %license LICENSES/MIT.txt
+%{_libdir}/qt5/qml/org/kde/bismuth/core/qmldir
+%{_libdir}/qt5/qml/org/kde/bismuth/core/libbismuth_core.so
 %{_libdir}/qt5/plugins/kcms/kcm_bismuth.so
+%{_libdir}/qt5/plugins/org.kde.kdecoration2/bismuth_kdecoration.so
 %{_datadir}/config.kcfg/bismuth_config.kcfg
-%{_datadir}/icons/hicolor/32x32/status/bismuth*
+%{_datadir}/kconf_update/bismuth_old_conf_ui.upd
+%{_datadir}/kconf_update/bismuth_shortcuts_category.upd
+%{_datadir}/kconf_update/bismuth_new_logger.upd
+%{_datadir}/kconf_update/bismuth_old_conf_ui.sh
+%{_datadir}/qlogging-categories5/bismuth.categories
+%{_datadir}/kservices5/kcm_bismuth.desktop
+%{_datadir}/kpackage/kcms/kcm_bismuth
+%{_datadir}/kpackage/kcms/kcm_bismuth/metadata.desktop
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/views
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/views/WorkspaceRules.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/views/WindowRules.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/views/MonocleOverlay.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/views/Layouts.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/main.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/components
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/components/RatioConfigSpinBox.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/components/PixelsConfigSpinBox.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/components/ConfigTextField.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/components/ConfigSpinBox.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/components/ConfigCheckBox.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/Behaviour.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/Appearance.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/metadata.json
+%{_datadir}/icons/hicolor/scalable/apps/bismuth.svg
 %{_datadir}/icons/hicolor/22x22/categories/bismuth-kcm.svg
 %{_datadir}/icons/hicolor/64x64/categories/bismuth-kcm.svg
-%{_datadir}/icons/hicolor/scalable/apps/bismuth.svg
-%{_datadir}/kconf_update/old_conf_ui.sh
-%{_datadir}/kconf_update/old_conf_ui.upd
-%{_datadir}/kpackage/kcms/kcm_bismuth
-%{_datadir}/kservices5/kcm_bismuth.desktop
 %{_datadir}/kwin/scripts/bismuth
+%{_datadir}/kwin/scripts/bismuth/contents
+%{_datadir}/kwin/scripts/bismuth/contents/ui
+%{_datadir}/kwin/scripts/bismuth/contents/ui/main.qml
+%{_datadir}/kwin/scripts/bismuth/contents/ui/popup.qml
+%{_datadir}/kwin/scripts/bismuth/contents/code
+%{_datadir}/kwin/scripts/bismuth/contents/code/index.mjs
+%{_datadir}/kwin/scripts/bismuth/metadata.desktop
```

Comment:
I can't verify it right now but I'm pretty sure you can add directories and it will add it's files